staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/redis/client.go

```diff
@@ -1,35 +1,121 @@
 package redis
 
 import (
+	"context"
+	"encoding/json"
+	"github.com/go-redis/redis/v8"
 	"github.com/kubeedge/Template/pkg/common"
-	"github.com/kubeedge/Template/pkg/global"
+	"k8s.io/klog/v2"
+	"os"
+	"strconv"
+)
+
+var (
+	RedisCli *redis.Client
 )
 
 type DataBaseConfig struct {
+	RedisClientConfig *RedisClientConfig
+}
+
+type RedisClientConfig struct {
+	Addr         string `json:"addr,omitempty"`
+	DB           int    `json:"db,omitempty"`
+	PoolSize     int    `json:"poolSize,omitempty"`
+	MinIdleConns int    `json:"minIdleConns,omitempty"`
 }
 
-func NewDataBaseClient() (global.DataBaseClient, error) {
-	return &DataBaseConfig{}, nil
+func NewDataBaseClient(config json.RawMessage) (*DataBaseConfig, error) {
+	configdata := new(RedisClientConfig)
+	err := json.Unmarshal(config, configdata)
+	if err != nil {
+		return nil, err
+	}
+	return &DataBaseConfig{RedisClientConfig: configdata}, nil
 }
 
 func (d *DataBaseConfig) InitDbClient() error {
-	//TODO implement me
-	panic("implement me")
+	var password string
```

Comment:
I have a question, is the password here plaintext or encrypted? Security issues may need to be considered.

Reply:
The password here is first encrypted and then mounted into the mapper through Secret.

Reply:
ok